design/apidsl/media_type.go

```diff
@@ -12,6 +12,7 @@ import (
 // Counter used to create unique media type names for identifier-less media types.
 var mediaTypeCount int
 
+// MediaType can be used in: ResponseTemplate, (also as a top level DSL)
```

Comment:
I was kind of following that pattern for the other top level DSL's, API was written like that but I think now maybe it looks better out of parentheses so I will change the others too.